### PR TITLE
[docs] correct typo in all versions of yb-admin

### PR DESCRIPTION
**Pull request by polarweasel**

### CODE REVIEW

**Review by aishwarya24 — APPROVED** [review]
LGTM

File: docs/content/latest/admin/yb-admin.md [hunk 1]
```diff
@@ -106,7 +106,7 @@ If you need to take a node down temporarily, but intend to bring it back up, you
 * If the node is down for less than 15 minutes, it will catch up through RPC calls when it comes back online.
```
Comment (ddhodge) [inline docs/content/latest/admin/yb-admin.md:106]: The list just above has ADD SERVER and REMOVE SERVER in all these

**Review by ddhodge — APPROVED** [review]
LGTM except comment

File: docs/content/latest/admin/yb-admin.md (hunk 1 above)
Comment (polarweasel) [reply docs/content/latest/admin/yb-admin.md:106]: Good catch! Fixed in the latest commit.

**Pull request merged**